Allow to change the schedule on run-time (#12)
hello-nrfcloud/backend: README.md

```

which will send simulated data to nRF Cloud.

## Shadow fetcher configuration

To customize the frequency of the shadow fetcher for each model, you can modify
the configuration using

```bash
./cli.sh set-shadow-fetcher-config <modelName> <value>
```

The value parameter can be specified in the following formats:

- interval
- interval:count
- interval:count, interval:count

> Here, `interval` refers to the duration in seconds for fetching the shadow, and
> `count` represents the number of iterations.

Reviewer: ```suggestion
Here, `interval` refers to the interval in seconds between fetching the shadow of a device, and
`count` represents the number of iterations.
```

Please provide an example, e.g. what would 17:42 mean technically?
